internal/host/validator.go

```diff
@@ -916,41 +916,57 @@ func (v *validator) sufficientOrUnknownInstallationDiskSpeed(c *validationContex
 	return ValidationFailure, "While preparing the previous installation the installation disk speed measurement failed or was found to be insufficient"
 }
 
-func (v *validator) hasSufficientNetworkLatencyRequirementForRole(c *validationContext) (ValidationStatus, string) {
-	if c.infraEnv != nil {
-		return ValidationSuccessSuppressOutput, ""
-	}
-	if len(c.cluster.Hosts) == 1 || c.clusterHostRequirements.Total.NetworkLatencyThresholdMs == nil || common.GetEffectiveRole(c.host) == models.HostRoleAutoAssign || hostutil.IsDay2Host(c.host) {
-		// Single Node use case || no requirements defined || role is auto assign
-		return ValidationSuccess, "Network latency requirement has been satisfied."
-	}
-	if len(c.host.Connectivity) == 0 {
-		return ValidationPending, "Missing network latency information."
-	}
-	s, hostLatencies, err := v.validateNetworkLatencyForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts, c.inventoryCache)
-	if s == ValidationFailure {
-		if err != nil {
-			return ValidationFailure, fmt.Sprintf("Error while attempting to validate network latency: %s", err)
+type hostTimingMetric struct {
+	otherHostName string
+	timingMetric  float64
+	timingSuffix  string
+}
+
+const MAX_HOST_TIMING_METRICS = 4
```

Comment:
Please put the const at the top of the file.
Additionally, the naming convention should be `camelCase`

Reply:
Done and done